packages/devp2p/src/eth/index.ts

```diff
@@ -134,12 +134,12 @@ export class ETH extends EventEmitter {
     const c = this._peer._common
 
     const peerForkHash = `0x${forkId[0].toString('hex')}`
-    const peerNextFork = new BN(forkId[1])
+    const peerNextFork = toType(forkId[1], TypeOutput.BigInt)
```

Comment:
in all of devp2p, since we know we are working with buffers, lets use bufferToBigInt rather than toType